server/test/unit/server/webservices/views/test_consumers.py

```diff
@@ -1009,6 +1060,19 @@ def test_create_consumer_profile_missing_param(self, mock_profile):
         self.assertEqual(response.error_data['property_names'], ['content_type'])
 
 
+class TestConsumerProfileSearchView(unittest.TestCase):
+    """
+    Test the ConsumerProfileSearchView.
+    """
+    def test_class_attributes(self):
+        """
+        Ensure that the ConsumerProfileSearchView has the correct class attributes.
+        """
+        self.assertEqual(ConsumerProfileSearchView.response_builder,
+                         util.generate_json_response_with_pulp_encoder)
```

Comment:
It does seem like maybe all the search views use this response builder. Maybe we can just drop that and have the superclass just use it so they don't all have to specify it. We could do that in a later PR...